data/pathfinder/alluria_publishing/remarkable_races/rr_races.lst

```diff
@@ -25,7 +25,7 @@ Zif											STARTFEATS:1	SIZE:M	MOVE:Walk,20																LANGBONUS:TYPE=Spo
 
 # Bestiary
 Mechanomancer									STARTFEATS:1	SIZE:L	MOVE:Walk,30																																						BONUS:STAT|STR|14	BONUS:STAT|DEX|2	BONUS:STAT|INT|6	BONUS:STAT|CHA|-4							BONUS:VAR|DarkvisionRange|60|TYPE=Base	BONUS:VAR|FireResistanceBonus,ElectricityResistanceBonus|20|TYPE=Resistance	BONUS:VAR|SlamAttacks|1	BONUS:WEAPONPROF=Slam|CRITRANGE|1													ABILITY:Internal|AUTOMATIC|Race Traits ~ Mechanomancer	ABILITY:Internal|AUTOMATIC|Slam	LEGS:2	HANDS:2																	DR:10/Magic		MONSTERCLASS:Construct:10			RACETYPE:Construct			RACESUBTYPE:Boggle							TYPE:Construct		CR:8			SOURCEPAGE:p.137	SPELLS:Innate|TIMES=1|CASTERLEVEL=6|Fireball
-Rift Swarm										STARTFEATS:1			MOVE:Walk,20				FACE:10	REACH:0																																																	BONUS:VAR|HasLowlightVision|1																																						ABILITY:Internal|AUTOMATIC|Race Traits ~ Rift Swarm												NATURALATTACKS:Swarm,Weapon.Melee,*1,4d6,SPROP=plus Poison (Fort DC 26)															RACETYPE:Outsider				RACESUBTYPE:Swarm								TYPE:Outsider		CR:14			SOURCEPAGE:p.140
+Rift Swarm										STARTFEATS:1			MOVE:Walk,20				MODIFY:Face|SET|10,10|PRIORITY=200																																																	BONUS:VAR|HasLowlightVision|1																																						ABILITY:Internal|AUTOMATIC|Race Traits ~ Rift Swarm												NATURALATTACKS:Swarm,Weapon.Melee,*1,4d6,SPROP=plus Poison (Fort DC 26)															RACETYPE:Outsider				RACESUBTYPE:Swarm								TYPE:Outsider		CR:14			SOURCEPAGE:p.140
 Gumpawomp										STARTFEATS:1	SIZE:L	MOVE:Walk,20																															BONUS:COMBAT|AC|7|TYPE=NaturalArmor																				BONUS:VAR|RaceSizeIsLong|1																				BONUS:WEAPONPROF=Tail|REACH|5														ABILITY:Internal|AUTOMATIC|Race Traits ~ Gumpawomp								LEGS:4	HANDS:2	NATURALATTACKS:Tail,Weapon.Melee.Bludgeoning,*1,2d6																		RACETYPE:Animal				RACESUBTYPE:Gumpawomp							TYPE:Animal			CR:1			SOURCEPAGE:p.141
```

Comment:
Why does this line (and some others in various places) still have an old "RaceSizeIsLong" variable?